privaterelay/settings.py

```diff
@@ -677,7 +707,9 @@ def _get_initial_middleware() -> list[str]:
                 # To allow testing the subscription flow on stage, we can set
                 # a custom plan ID via an environment variable:
                 "id": BUNDLE_PLAN_ID_US,
-                "price": "$?.??",
+                # TODO: Insert correct price
+                "price": -1337,
```

Comment:
Just to confirm - "Monthly" and "yearly" refer to the billing period **only** for Bundle pricing, not for Phone?

Reply:
No, it's the billing period for everything - so the prices for the yearly plans for regular Premium and phones are monthy prices as well, but they're billed yearly.

Reply:
Got it, thanks for clearing up the confusion